serializable/helpers.py

```diff
@@ -205,7 +205,7 @@ def deserialize(cls, o: Any) -> datetime:
 
             # Ensure any milliseconds are 6 digits
             # Background: py<3.11 supports six or less digits for milliseconds
-            v = re_sub(r'\.(\d{1,6})', lambda m: f'.{int(m.group()[1:]):06}', v)
```

Comment:
this old implementation had so many flows.

besides the bug you try to fix, it would also convert `'1.337'` to `'1.000337'`